collectd/files/default_collectd.conf

```diff
@@ -0,0 +1,26 @@
+# /etc/default/collectd
+
+# 0: start collectd on boot, 1: do not start collectd on boot
+# default: 0
+DISABLE=0
+
+# 0: start collectd in stand-alone mode, 1: monitor collectd using collectdmon
+# default: 1
+USE_COLLECTDMON=1
+
+# number of seconds to wait for collectd to shut down
+# default: 30
+MAXWAIT=30
+
+# 0: do not enable core-files, 1: enable core-files ... if collectd crashes
+# default: 0
+ENABLE_COREFILES=0
+
+# For Java plugin
+{%- set java_ld_library_path = '' %}
+{%- set java_lib = salt['cmd.run']('type java >/dev/null 2>&1 && find /usr/lib/jvm -name libjvm.so -type f') %}
+{%- if java_lib|length > 0 %}
+  {%- set java_path = '/'.join(java_lib.split('/')[:-1]) %}
+  {%- set java_ld_library_path = 'export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:' ~ java_path %}
+{%- endif %}
+{{ java_ld_library_path }}
```

Comment:
You could do

```
{%- if java_ld_library_path != '' %}

# For Java plugin
{{ java_ld_library_path }}
{%- endif %}
```

to avoid displaying `For Java plugin` followed by an empty line when there's no Java installation on the machine.